ams/server/src/main/java/com/netease/arctic/server/table/executor/SnapshotsExpiringExecutor.java

```diff
@@ -258,31 +255,34 @@ public static void expireSnapshots(
     LOG.info("to delete {} files, success delete {} files", toDeleteFiles.get(), deleteFiles.get());
   }
 
-  public static Snapshot getClosestExpireSnapshot(UnkeyedTable changeTable, long ttl) {
+  public static Snapshot getClosestExpireSnapshot(UnkeyedTable changeTable, long ttlPoint) {
     if (changeTable.snapshots() == null) {
       return null;
     }
     return Streams.stream(changeTable.snapshots())
-        .filter(snapshot -> snapshot.timestampMillis() <= ttl)
-        .min(Comparator.comparingLong(snapshot -> Math.abs(ttl - snapshot.timestampMillis())))
+        .filter(snapshot -> snapshot.timestampMillis() < ttlPoint)
+        .min(Comparator.comparingLong(snapshot -> Math.abs(ttlPoint - snapshot.timestampMillis())))
```

Comment:
We can simplify the code and just use the `snapshot.timestampMillis()` to compare.

Reply:
I get it.